command/ssh/checkHost.go

```diff
@@ -61,7 +65,27 @@ func checkHostAction(ctx *cli.Context) error {
 		return err
 	}
 
-	resp, err := client.SSHCheckHost(ctx.Args().First())
+	id, err := ca.LoadDefaultIdentity()
+	if err != nil {
+		return errors.Wrap(err, "error loading the deault x5c identity")
```

Comment:
we should probably return something add something like `please run 'step ssh login <identity>'`